Dockerfile

```diff
@@ -11,7 +11,8 @@ RUN apt-get update \
         vim \
     && curl -sL https://deb.nodesource.com/setup_10.x | bash - \ 
     && apt-get install -y --no-install-recommends nodejs \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
+    && npm i -g npm@7.6.3
```

Comment:
let's do this the same way as you have done for Portal, put `npm` to `7.8.0` and set `maxsockets` value